src/seedu/addressbook/commands/FindByPhoneCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.Phone;
+import java.util.*;
+import java.lang.String;
+
+public class FindByPhoneCommand extends Command{
```

Comment:
Better if you can add s header comment for the class explaining the role of it to make it more understandable for the reader.